tests/suites/0_stateless/18_rbac/20_0012_privilege_access.result

```diff
@@ -18,7 +18,8 @@ test -- select
 1
 1
 1
-1
+test -- select view
+Error: APIError: ResponseError with 1063: Permission denied, privilege [Select] is required on 'default'.'default2'.'v_t20_0012' for user 'test-user'@'%' with roles [public,test-role1,test-role2]
```

Comment:
This modify generates a break change.

Before this pr, user can query view even though he does not have privilige about any table.

Reply:
no, i don't think this `user can query view even though he does not have privilige about any table` was the behavior before, this pr is not actually affected the related logic at all.